en/flight_modes/README.md

```diff
@@ -416,7 +416,7 @@ Key for understanding the table is as follows:
 
 Symbol | Description
 --- | ---
-M | Manual control via RC sticks. RC input is sent directly to the output mixer.
+M | Manual control via RC sticks. RC input is sent directly to the output mixer. 

```

Comment:
Many places say something like "is sent directly to the output mixer.". My understanding is that this is correct - you might still refer to it as an output mixer.

Would be be OK/better off writing this like "is sent directly the the control allocator"?

Reply:
You can still use mixer here. Or `output allocation`.

Reply:
Thanks. I prefer "output allocation". Mostly because the more we are consistent with terminology, the less likely we are to have questions about "where did the mixer go" etc.
I've changed it globally-ish via search replace in this Pr.